src/transformers/__init__.py

```diff
@@ -775,6 +775,8 @@
     "models.vit_msn": ["ViTMSNConfig"],
     "models.vitdet": ["VitDetConfig"],
     "models.vitmatte": ["VitMatteConfig"],
+    "models.vitpose": ["ViTPoseConfig"],
+    "models.vitpose_backbone": ["ViTPoseBackboneConfig"],
```

Comment:
```suggestion
    "models.vitpose": ["VitPoseConfig"],
    "models.vitpose_backbone": ["VitPoseBackboneConfig"],
```
I think we can go for `Vit` everywhere

Reply:
@NielsRogge I think ViT is correct since it is abbreviation. (Other transformers are using also ViT)

Reply:
https://github.com/huggingface/transformers/blob/cff06aac6fad28019930be03f5d467055bf62177/src/transformers/__init__.py#L384
https://github.com/huggingface/transformers/blob/cff06aac6fad28019930be03f5d467055bf62177/src/transformers/__init__.py#L470
https://github.com/huggingface/transformers/blob/cff06aac6fad28019930be03f5d467055bf62177/src/transformers/__init__.py#L621